src/pages/productPage/productPage.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import "../home/home.css";
+import { Footer, Navbar } from '../../components/allComponents';
+import "./productPage.css";
+import { ProductListing } from '../../components/productListing/productListing';
+import { Sidebar } from '../../components/sidebar/sidbar';
```

Comment:
suggestion: create index file in components and export all components from it

Reply:
i have updated it